src/.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: node_js
+node_js:
+  - "node"
+notifications:
+  email: false
+addons:
+  apt:
+    packages:
+    - libgconf-2-4
+cache:
+  npm: true
+  directories:
+    - ~/.npm
+    - ~/.cache
+  override:
+    - npm ci
+    - npm run cy:verify
+install:
+  - npm ci
+script:
+  - npm start &
+  - npm run start:ci &
+  - cypress run --record --key 86ce623b-9bbc-4993-8dca-f3535c528333
```

Comment:
I believe we will add a deploy key here as well which is where we will put vercel

Reply:
Thanks fam!